### PR TITLE
Use Dot notation

### PR DESCRIPTION
**Pull request by amitaibu**

### CODE REVIEW

File: Test/Controller/ReservationsSpec.hs [hunk 1]
```diff
@@ -64,18 +64,18 @@ tests = aroundAll (withIHPApp WebApplication config) do
...
-                reservation <- fetch (get #reservationId reservationJob)
+                reservation <- fetch reservationJob.reservationId
                 -- Seat not assigned yet.
-                get #seatNumber reservation `shouldBe` 0
+                reservation.seatNumber `shouldBe` 0
 
 
                 -- Process job.
-                let frameworkConfig = getFrameworkConfig ?context
+                let frameworkConfig = ?context.frameworkConfig
```
Comment (amitaibu) [inline Test/Controller/ReservationsSpec.hs:74]: @mpscholten trying to fix the test with the new syntax to get the `frameworkConfig`, but when I try to [run it](https://github.com/Gizra/ihp-simple-seat-reservation#test) I get an error

image

Comment (mpscholten) [reply Test/Controller/ReservationsSpec.hs:74]: Hm, could you do an `import IHP.Controller.RequestContext ()`?

Comment (amitaibu) [reply Test/Controller/ReservationsSpec.hs:74]: Doesn't work - same error

Comment (mpscholten) [reply Test/Controller/ReservationsSpec.hs:74]: Try `import IHP.Controller.RequestContext` without the `()` at the end. Locally this got it working 👍

**Pull request merged**